handbook/engineering/search/iteration_log.md

```diff
@@ -31,6 +29,28 @@ This document contains the goals and work log for the search team's [2-week iter
     - RFC published.
 - **Work log:**
     - YYYY-MM-DD: $UPDATE
+    
+### Structural search for monorepos
+- **Owner(s):** Rijnard, Camden
+- **Outcomes:**
+    - Make structural search work better for monorepos. Currently monorepos can time out for structural search purely because it takes too long to copy the data to search. The outcome focuses on architectural changes to avoid unnecessary data zipping and copying to our searcher worker.
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Search rules
+- **Owner(s):** Rijnard
+- **Outcomes:**
+    - RFC published.
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
```

Comment:
@rvantonder I added this placeholder outcome for your work on search rules. Can you review & make sure it's accurate?

Reply:
This is fine, I will update a link to the state of the WIP RFC by the next work log update.